multiplayer/src/components/GamePage.tsx

```diff
@@ -31,9 +31,12 @@ export default function Render(props: GamePageProps) {
                 </div>
             )}
             <div
-                className={`tw-flex tw-flex-col tw-items-center ${
-                    state.gameState?.gameMode === "playing" ? "" : "hidden"
-                }`}
+                className="tw-flex tw-flex-col tw-items-center tw-grow tw-pb-4"
+                style={
```

Comment:
reason to move this to a style is because tailwind jit does not work with non-string literals like template strings